onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -101,19 +102,19 @@ class ConvActivationSelector : public NodeSelector {
 
       return false;
     };
-
+    const auto& domain = node.Domain();
     if (!ConvFusionDataTypeCheck(node)) {
       return std::nullopt;
     }
 
     // check EP type and activation
     if (node_ep == kCudaExecutionProvider || node_ep == kRocmExecutionProvider) {
-      if (!graph_utils::IsSupportedOptypeVersionAndDomain(*next_node, "Relu", {6, 13, 14})) {
+      if (!graph_utils::IsSupportedOptypeVersionAndDomain(*next_node, "Relu", {6, 13, 14}, domain)) {
```

Comment:
Should this be using the domain from the next_node and not the current node? 

e.g. if I have a Conv in the internal NHWC domain, the Relu will be in the ONNX domain and this check will reject the fusion.

Reply:
Dropped domain argument.